db_json_util.py

```diff
@@ -74,6 +76,35 @@ def main() -> None:
                     else:
                         print(response.text)
 
+# export data using the API
+        if args.export_data:
+            if not os.path.exists('json_exports'):
+                os.makedirs('json_exports')
+            if args.json_chains:
+                url_format = args.url + '/all/chains'
+                response = requests.get(url_format)
+                if response.status_code == 200:
+                    data_chains = response.json()
+                    export_name = "json_exports/chains_exported.json"
```

Comment:
Instead of using a hard coded file path we should use the `path_chains` variable, which is constructed from what's given to the `ArgumentParser`. Basically, it should be up to the user to decide where the files should go.

Reply:
But this will override the original data in the JSON file, is this what is intended?

Reply:
Ohh... I understand now. The user defines the path.

I'll make these changes